change/@fluentui-react-context-selector-077a755a-703f-4be3-92c8-8fd47e3614e2.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
I think this will need to be prerelease, and you're missing a change file for accordion.